src/main/resources/google_checks.xml

```diff
@@ -251,7 +251,8 @@
         </module>
         <module name="NoWhitespaceBefore">
             <property name="tokens"
-             value="COMMA, SEMI, POST_INC, POST_DEC, DOT, ELLIPSIS, METHOD_REF"/>
+             value="COMMA, SEMI, POST_INC, POST_DEC, DOT, ELLIPSIS,
+                    LABELED_STAT, METHOD_REF"/>
```

Comment:
please update google IT  input for this Check.
We need to have in Inputs all cases to immediately see affect on google is any regression is about to happen.